controllers/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -15,7 +16,7 @@ class UserController(userService: UserService) {
   private val getUser: Route = get {
     parameter('email.as[String]) { email =>
       onComplete(userService.getUsersByEmail(email)) {
-        case Success(r) => complete(r)
+        case Success(usersWithCredentials) => complete(usersWithCredentials.map(User.fromUserWithCredentials))
```

Comment:
I think we can add `apply(user: UserWithCredentials): User` method to the object User and some implicit method like implicit `def fromUserWithCredentials(user: UserWithCredentials): User = User(user)`
Or we can create some converting from Seq[UserWithCredentials] to Seq[User] with implicits
I think it would be better for the project and coders.
But it`s up tou you

Reply:
Thanks for comment. 
1) I think apply method may lead to worse understanding of code. Also previous implementation contained fromUser method.
2) During mapping smth we should clearly specify method otherwise compiler will not be able to understand  what type we should get.
3) For creating converting from Seq[UserWithCredentials] to Seq[User] we also should specify type clearly Seq[User] to compilator will be able to eliminate type mismatch.

@myazinn what do you think?

Reply:
I don't see why we will experience compile issues.
`User.fromUserWithCredentials(userWithCredentials)` can be converted to `User.apply(userWithCredentials)` which can be further converted to `User(userWithCredentials)`. At which do you think the problem will arise?
In this particular case we can go even further and replace it with underscore `usersWithCredentials.map(User(_))`. It will work just fine.

Reply:
> I don't see why we will experience compile issues.
> `User.fromUserWithCredentials(userWithCredentials)` can be converted to `User.apply(userWithCredentials)` which can be further converted to `User(userWithCredentials)`. At which step do you think the problem will arise?
> In this particular case we can go even further and replace it with underscore `usersWithCredentials.map(User(_))`. It will work just fine.

Ok. I'll add apply method. I just wrote the map method requires clearly specify converting even if we use implicit methods.

Reply:
Done.